var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -49,10 +49,14 @@ class Mfem(Package, CudaPackage, ROCmPackage):
     # other version.
     version('develop', branch='master')
 
+    version('4.3.0',
+            sha256='3a495602121b986049286ea0b23512279cdbdfb43c15c42a1511b521051fbe38',
+            url='https://bit.ly/mfem-4-3', extension='tar.gz',
+            perferred=True)
```

Comment:
Spack already defaults to the latest stable release. If you remove `preferred=True` the diff will be smaller each time you add a new version.

Reply:
Okay, I'll remove it (it is also misspelled right now).